org.tribler.Tribler.yaml

```diff
@@ -0,0 +1,64 @@
+id: org.tribler.Tribler
+command: tribler
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-download
+  - --persist=.Tribler
+  - --own-name=org.kde.*
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: tribler
+    buildsystem: simple
+    build-commands:
+      - ar x tribler.deb
+      - tar xf data.tar.xz
+      - mkdir -p /app/bin
+      - install tribler /app/bin
+      - install -Dm644 usr/share/applications/tribler.desktop /app/share/applications/org.tribler.Tribler.desktop
+      - |
+        for size in 64 128 256 512; do
+          install -Dm644 tribler_${size}.png /app/share/icons/hicolor/${size}x${size}/apps/org.tribler.Tribler.png || exit 1
+        done
+      - mv usr/share/tribler /app/share
+      - install -Dm644 README.md -t /app/share/tribler
+      - install -Dm644 org.tribler.Tribler.metainfo.xml -t /app/share/metainfo
+    sources:
+      - type: file
+        url: https://github.com/Tribler/tribler/releases/download/v7.8.0/tribler_7.8.0_all.deb
+        sha256: 22ac6be8a06024dd58f1759c89816100461543f593056f697b2c40d851df85a4
+        dest-filename: tribler.deb
+      - type: file
+        path: README.md
+      - type: dir
+        path: icons
```

Comment:
Are these files unavailable in the deb package?

Reply:
Unfortunately yes, there is only a very low resolution png in the deb.